spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] It will be nice to change `Comment` to `User` since we are testing the `User` class model.

Reply:
This has been resolved